src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -1081,7 +1081,7 @@ void MergeTreeData::PartLoadingTree::add(const MergeTreePartInfo & info, const S
             else if (!prev_info.isDisjoint(info))
             {
                 throw Exception(ErrorCodes::LOGICAL_ERROR,
-                    "Part {} intersects previous part {}. It is a bug!",
+                    "Part {} intersects previous part {}. It is a bug or a result of manual intervention in the server or ZooKeeper data",
```

Comment:
This is total bullshit. You've read my bug report [here](https://github.com/ClickHouse/ClickHouse/issues/50178) which does not say anything about manual intervention or Zookeeper. Yet somehow decided to improve the message.

Reply:
> some extra disks were not mounted properly

There's nothing ClickHouse could do about this. From the server's point of view, unmounting some disks and starting clickhouse-server is like manually [re]moving the data from these disks. There's no real difference between "manual intervention" and "some extra disks were not mounted properly", in both cases it's not ClickHouse's problem. You should not expect that a DBMS will continue working as usual when its data on disks is broken due to third-party reasons.

Reply:
I've lost data and CH was broken just because some disks were not mounted in time, not even broken or corrupted. Surely this can be handled better.